RZA-250021: Refactor and Optimization of existing Pages

1046prt opened the pull request:
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Browser:
- Device:
- Toolchain:

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

Ayu-Ty committed:
added selection, modified constants and quote designer

1046prt committed:
Make change in title of draw page

1046prt committed:
fix the bugs of scroll bar as when pop section is open and also make changes in its design

Rupesh-Rathore committed:
Text animation added

Rupesh-Rathore committed:
swap button added

Rupesh-Rathore committed:
UI and Animation updated to make more responsive and appealing

Suresh-Chelani committed:
add andhra pradesh, responsiveness

Suresh-Chelani committed:
add modified files

1046prt committed:
I have fixed position of continue message position to top instead of bottom

Ayu-Ty committed:
modified games.css, selection.astro, index.astro, and constants.ts

1046prt committed:
Modified the position of "Press any key" text and also modified design of word section

Ayu-Ty committed:
added back buttons to canvas and advanced

Azmat01 committed:
Add custom colors and new button states in crossword styles; update footer and globals for responsive design

Azmat01 committed:
add modified files

1046prt committed:
update the UI of page based on abcdkbd.com

1046prt committed:
modified the UI of capital page and also comment out the previous code

1046prt committed:
modify UI of page for light mode also and enhance some Feature

Rupesh-Rathore committed:
CREATED ONE-LINE UNIT CONVERTER

Rupesh-Rathore committed:
updated responsivity

Rupesh-Rathore committed:
REMOVED GRID BADES UNIT LAYOUT AND CREATED TIMELINE BASED LAYOUT

chhaviS04 committed:
Fixed the issue of increasing of scores on rewrite

chhaviS04 committed:
updated month section

Suresh-Chelani committed:
add correct text with responsiveness

Suresh-Chelani committed:
add arunachal pradesh map using functionality of svg events

Rupesh-Rathore committed:
Smoothen the timeline animation nad responsiveness

1046prt committed:
change the search icon using svg image

1046prt committed:
modified the UI of capital page for dark mode and light mode also

Suresh-Chelani committed:
add all the districts of maps

Azmat01 committed:
Enhance puzzle UI with improved button styles and animations

1046prt committed:
Change the font style with reference from abcdkbd.com

1046prt committed:
Change the Message for "Press any key to continue" and modify the bugs in footer

Ayu-Ty committed:
Modified designer tools, added back button, and implemented keyboard interaction in Keyboard game

Rupesh-Rathore committed:
minor adjustments

Rupesh-Rathore committed:
Changed the layout of VEDIC page and created the tab-pane  layout

Rupesh-Rathore committed:
added labels to converter and updated timeline UI

1046prt committed:
Resolved the raised issue of both pages (i.e Capital and Draw)

1046prt committed:
Modify the search functionality with the use of debouncing functionality

Suresh-Chelani committed:
increase the size of map, add some style for better view in state component

Rupesh-Rathore committed:
STICKY TITLE OF TIMELINE

Ayu-Ty committed:
added numbers to keyboard game

Azmat01 committed:
Enhance button styles and interactions in Puzzle component; update CSS variables for improved theming

1046prt committed:
Modify the page title as originally was and also align continue message

1046prt committed:
Modify card alignment and add highlight in search

chhaviS04 committed:
Added Sun Signs Tile

chhaviS04 committed:
updated month section

chhaviS04 committed:
Added Greek Alphabets

Suresh-Chelani committed:
add map view options and change hover effect

Suresh-Chelani committed:
added more tiles in map

chhaviS04 committed:
Improved Layout

Rupesh-Rathore committed:
ADDED VEDIC TIME JOURNEY TO TAB_PANE FORMAT , Updated the UI of VEDIC page

Rupesh-Rathore committed:
MODIFICATION IN VEDIC

Rupesh-Rathore committed:
MODIFICATION IN UI OF KANNADA PAGE

Rupesh-Rathore committed:
FIXED THE TITHI CLOCK NOT WORKING BUG

1046prt committed:
Fix minute color change in card section

Ayu-Ty committed:
implemented reset and setting buttons, removed designer tools, improved keyboard game's ui

Rupesh-Rathore committed:
improved the UI of VEDIC converter

Rupesh-Rathore committed:
ADDED GUJARATI TO KANNADA PAGE

Rupesh-Rathore committed:
ADDED MARATHI AND PUNJABI(GURMUKHI) to KANNADA PAGE

chhaviS04 committed:
Modified Alphabet section

Rupesh-Rathore committed:
ADDED PUNJABI AND MARATHI TO KANADA PAGE.
created unselectable placeholder for options in dropdowns

1046prt committed:
Refactor location card styles and animations
Consolidate color variations and streamline responsive design

Ayu-Ty committed:
added back buttons, made ui changes

Azmat01 committed:
Refactor styles in stats and settings components; adjust button sizes and layout for better responsiveness

Azmat01 committed:
Merge branch 'feature/RZA-250021' of https://github.com/recursivezero/abcd into feature/RZA-250021

chhaviS04 committed:
fixed backspace input issue

Ayu-Ty committed:
Merge branch 'feature/RZA-250021' of https://github.com/recursivezero/abcd into feature/RZA-250021

Ayu-Ty committed:
added comment

Rupesh-Rathore committed:
All of MAY 5 chaos resolved

xkeshav committed:
[RZA-250021]: fix svg files and revert deleted files

xkeshav committed:
[RZA-250021]: fix lint issues

xkeshav commented:
fix the hints except keyCode one

```log
src/components/kannadaHindi.astro:223:9 - warning ts(6133): 'languages' is declared but its value is never read.

223   const languages = [
            ~~~~~~~~~

src/components/map/District.astro:90:9 - warning astro(4000): This script will be treated as if it has the `is:inline` directive because it contains an attribute. Therefore, features that require processing (e.g. using TypeScript or npm packages in the script) are unavailable.

See docs for more details: https://docs.astro.build/en/guides/client-side-scripts/#script-processing.

Add the `is:inline` directive explicitly to silence this hint.

90 <script define:vars={{ state }}>
           ~~~~~~~~~~~

src/pages/advanced.astro:208:5 - warning astro(4000): This script will be treated as if it has the `is:inline` directive because it contains an attribute. Therefore, features that require processing (e.g. using TypeScript or npm packages in the script) are unavailable.

See docs for more details: https://docs.astro.build/en/guides/client-side-scripts/#script-processing.

Add the `is:inline` directive explicitly to silence this hint.

208     src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.4.1/html2canvas.min.js"
        ~~~

src/pages/panel.astro:195:53 - warning ts(6133): 'e' is declared but its value is never read.

195         this.#cellRange.addEventListener("change", (e) => {
                                                        ~
src/pages/panel.astro:152:35 - warning ts(6385): 'keyCode' is deprecated.

152           const { charCode, code, keyCode, key } = e;
                                      ~~~~~~~
src/pages/panel.astro:152:19 - warning ts(6385): 'charCode' is deprecated.

152           const { charCode, code, keyCode, key } = e;
                      ~~~~~~~~

src/pages/vedic.astro:139:32 - warning ts(6133): 'index' is declared but its value is never read.

139           timeUnits.map((unit, index) => (
                                   ~~~~~
src/pages/vedic.astro:70:13 - warning ts(6133): 'dateInHindi' is declared but its value is never read.

70       const dateInHindi = new Intl.DateTimeFormat("hi", styleOption);
               ~~~~~~~~~~~
src/pages/vedic.astro:67:13 - warning ts(6133): 'dateInEnglish' is declared but its value is never read.

67       const dateInEnglish = new Intl.DateTimeFormat("en-us", fullEnglish);
               ~~~~~~~~~~~~~
src/pages/vedic.astro:12:13 - warning ts(6133): 'options' is declared but its value is never read.

12       const options = {
               ~~~~~~~
src/pages/vedic.astro:11:13 - warning astro(4000): This script will be treated as if it has the `is:inline` directive because it contains an attribute. Therefore, features that require processing (e.g. using TypeScript or npm packages in the script) are unavailable.

See docs for more details: https://docs.astro.build/en/guides/client-side-scripts/#script-processing.

Add the `is:inline` directive explicitly to silence this hint.

11     <script type="text/javascript" async>
               ~~~~

src/pages/alphabets/[alphabet]/playground.astro:203:22 - warning ts(6385): 'keyCode' is deprecated.

203         const { key, keyCode } = e;
                         ~~~~~~~

src/pages/map/[...state].astro:17:39 - warning ts(6133): 'svg' is declared but its value is never read.

17 const { name, dance, festival, image, svg, wikiLink, description } = stateInfo;
                                         ~~~

src/pages/varnmala/english.astro:64:24 - warning ts(6385): 'keyCode' is deprecated.

64           const { key, keyCode } = e;
                          ~~~~~~~

src/pages/varnmala/months.astro:239:14 - warning ts(6133): 'openCalendarView' is declared but its value is never read.

239     function openCalendarView(monthIndex, displayMonthName) {
                 ~~~~~~~~~~~~~~~~
src/pages/varnmala/months.astro:176:5 - warning astro(4000): This script will be treated as if it has the `is:inline` directive because it contains an attribute. Therefore, features that require processing (e.g. using TypeScript or npm packages in the script) are unavailable.

See docs for more details: https://docs.astro.build/en/guides/client-side-scripts/#script-processing.

Add the `is:inline` directive explicitly to silence this hint.

176     define:vars={{
        ~~~~~~~~~~~

src/pages/varnmala/reader.astro:163:29 - warning ts(6385): 'which' is deprecated.

163       const { key, keyCode, which, code } = e;
                                ~~~~~
src/pages/varnmala/reader.astro:163:20 - warning ts(6385): 'keyCode' is deprecated.

163       const { key, keyCode, which, code } = e;
                       ~~~~~~~
src/pages/varnmala/reader.astro:151:32 - warning ts(6133): 'e' is declared but its value is never read.

151         audioPlayer.onerror = (e) => {
                                   ~
src/pages/varnmala/reader.astro:51:58 - warning ts(6133): 'displayText' is declared but its value is never read.

51     function setSpeaker(name: string | null | undefined, displayText: string) {
                                                            ~~~~~~~~~~~
src/pages/varnmala/reader.astro:44:11 - warning ts(6133): 'selectedVoiceText' is declared but its value is never read.

44     const selectedVoiceText = document.querySelector("#selected-voice-text") as HTMLSpanElement;
             ~~~~~~~~~~~~~~~~~

Result (94 files): 
- 0 errors
- 0 warnings
- 21 hints
```

Azmat01 committed:
updated css both setting and crossword

1046prt committed:
Append the image path

1046prt committed:
Add icon for changing the image

1046prt committed:
Image are now visible on local host also

1046prt committed:
Appends all the images into assets

Rupesh-Rathore committed:
changes

Rupesh-Rathore committed:
Merge branch 'feature/RZA-250021' of https://github.com/recursivezero/abcd into feature/RZA-250021

1046prt committed:
Modify the UI of page

1046prt committed:
change some functionlaity of page

Suresh-Chelani committed:
fix; backup all the commit of map section

Rupesh-Rathore committed:
fixed all the removed/deleteed code

xkeshav reviewed (changes requested):
CSS changes

1046prt committed:
Removed the animation effect

1046prt committed:
Modify the Image icon functionality

Ayu-Ty committed:
added setting button, and reset buttons

1046prt committed:
Modify the svg image portion

xkeshav committed:
[RZA-250021]: added horizontal scroll;snapping on index page

xkeshav committed:
[RZA-250021]: fix lint errors on draw and Capital page

xkeshav committed:
[RZA-250021]: update workspace file

recursivezero reviewed (commented):
need to update more

Suresh-Chelani committed:
added the tiles of bards and dance

Suresh-Chelani committed:
added a JSON file containing mock data for map categories

Suresh-Chelani committed:
added back buttons to all sections of map

Suresh-Chelani committed:
added Read More feature in state description

Azmat01 committed:
refactor styles for puzzle and settings components; update button interactions and tooltip visibility

xkeshav committed:
[RZA-250021]: added code workspace in gitignore

chhaviS04 committed:
added greek and geek alphabets

chhaviS04 committed:
modified months section

chhaviS04 committed:
Added SunSigns

1046prt committed:
Icons for DrawCard Page

1046prt committed:
Adding the new page to Home

1046prt committed:
Adding component to DrawCard

1046prt committed:
Adding Queue Card Animation to DrawCard

1046prt committed:
Added some font color and directory of DrawCard Page

1046prt committed:
Add icon for changing font colors

1046prt committed:
Enhance and add functionality to change the font colors

chhaviS04 committed:
modified alphabets

Rupesh-Rathore committed:
Kannada__modified

chhaviS04 committed:
Fixed the top menu for mobile view

Azmat01 committed:
Update hint tooltip styles for better visibility

1046prt committed:
Update the image link and also fix some bugs

Suresh-Chelani committed:
remove the paintmaps text

Suresh-Chelani committed:
added responsiveness to the state map

Suresh-Chelani committed:
update the filename

Suresh-Chelani committed:
fix the loading error

Rupesh-Rathore committed:
kannada page code refactored and dropdown option css modified

Rupesh-Rathore committed:
made the language selecter stick to the top

Rupesh-Rathore committed:
Kannada page title  & help description changed

1046prt committed:
Align icons next to container in mobile view

1046prt committed:
Fix some bugs and seperate out css file

1046prt committed:
Fix some bugs and seperate out css file

1046prt committed:
CSS files for DrawCard component page

Ayu-Ty committed:
improved ui

Azmat01 committed:
Enhance search functionality and update PDF download button styles

chhaviS04 committed:
changed alphabets and glossary

Rupesh-Rathore committed:
Description of help updated and UI improved

1046prt committed:
changed the responsiveness for mobile and tab view

1046prt committed:
change the title of page and modifies some UI component

1046prt committed:
Improved UI of Card page components

chhaviS04 committed:
Added Flags,colours and seperated Alphabets

chhaviS04 committed:
added vowels

1046prt committed:
Modify the URL of Cards page and change the path of images to public

chhaviS04 committed:
updated flip icon for colours and flag page

1046prt committed:
changed the file path of image source and also font style

Ayu-Ty committed:
changed file names

Azmat01 committed:
Refactor clan page styles and card component functionality

- Simplified CSS for clan tree layout, enhancing responsiveness and visual consistency.
- Improved search functionality in the clan page, allowing for better text highlighting and matching.
- Updated card component to streamline data handling and improve user interaction with expanded views.
- Enhanced accessibility features, including ARIA attributes for better screen reader support.
- Removed redundant code and optimized event listeners for performance improvements.

Suresh-Chelani committed:
feat(svg-map): remove redundant SVGs and add zoom, pan, and reset functionality

chhaviS04 committed:
changed alignment

chhaviS04 committed:
modified for playground option

Rupesh-Rathore committed:
Modified vedic.json to display more relevant and important data

1046prt committed:
Added virtual keyboard for tab/mobile view and fixed some bugs

1046prt committed:
Refactor the responsiveness of keyboard in smaller devices

Suresh-Chelani committed:
feat(index.astor): added dark and light mode

chhaviS04 committed:
improved layout

Rupesh-Rathore committed:
malayalam-corrected, bangali-corrected, assamese-corrected, tamil-added, telugu-added, odia-added, dogri-added

1046prt committed:
Added slider view in the cards page

1046prt committed:
Refactor the URL of cards page components

1046prt committed:
Refactor the Image container size and its UI

1046prt committed:
removed cards folder

1046prt committed:
Added card page

Azmat01 committed:
Refactor color variables and styles for card and societree components

chhaviS04 committed:
added new pages waqt,morse code also updated colours and roman numerals

chhaviS04 committed:
created a page for hindi under Alphabets

Azmat01 committed:
Update card styles and improve surname card hover effects

1046prt committed:
Refactor the size of font problem and also add functionality of A11Y features

1046prt committed:
Refactor the css code and also remove the KeyPress event code for better animation experience

1046prt committed:
Align the arrow button to center of content

chhaviS04 committed:
Added Morse Alphabet

Suresh-Chelani committed:
added the state statistics

chhaviS04 committed:
Added a new page for Alphabet Writing Animation

Azmat01 committed:
Refactor CSS for responsive design and improve layout consistency

1046prt committed:
Refactor all 3 changes on capital page

Ayu-Ty committed:
improved ui

Ayu-Ty committed:
refactored canvas page

Suresh-Chelani committed:
fix the light & dark mode switching

Suresh-Chelani committed:
remove the outlines with added same colors

Suresh-Chelani committed:
fix back button issue

Suresh-Chelani committed:
Remove outlines from svg map, update the state statistics

1046prt committed:
Refractor the back button to left over page

1046prt committed:
Shifted the css of slider to assets folder

Rupesh-Rathore committed:
vedic time journey new UI

Rupesh-Rathore committed:
Responsiveness fixed,  arranged the dropdown options in ascending downwward

Rupesh-Rathore committed:
kannad name to indic

Rupesh-Rathore committed:
back-button added

Rupesh-Rathore committed:
added back button

Suresh-Chelani committed:
fix light mode issue

Rupesh-Rathore committed:
added ather languages

chhaviS04 committed:
Refactored glossary,varnmala,numbers ,alphabet pages

Azmat01 committed:
refactor: update color variables and improve dark mode styles in CSS files

Azmat01 committed:
Merge branch 'feature/RZA-250021' of https://github.com/recursivezero/abcd into feature/RZA-250021

Ayu-Ty committed:
ui improvements

xkeshav committed:
[RZA-250021]: fixed Gujarat SVG

Suresh-Chelani committed:
fix redundant code

Ayu-Ty committed:
added back

chhaviS04 committed:
Fixed the position of back button

1046prt committed:
Refactor the URL of draw page

1046prt committed:
Refactor the back button position

Rupesh-Rathore committed:
removed the unused code

Azmat01 committed:
Update body border color to black and refine card styles for print

chhaviS04 committed:
Removed duplicate logics

chhaviS04 committed:
Added Speak and draw page under varnmala

chhaviS04 committed:
Added Header

chhaviS04 committed:
Removed extra Vowels

chhaviS04 committed:
Fixed the alignment of morse code

chhaviS04 committed:
refined code

1046prt committed:
Refactor the title of cards page and fix the back button

Rupesh-Rathore committed:
Created the languageLetters,ts and removed the duplicacy

Rupesh-Rathore committed:
added back button

xkeshav committed:
[RZA-250021]: added docsify pages

Rupesh-Rathore committed:
Tithi corrected and disclaimer added

xkeshav reviewed (changes requested):
do changes

chhaviS04 committed:
Added Birds and colours in Alphabet section

Suresh-Chelani committed:
remove: redundant file

xkeshav commented on src/assets/styles/varnmala/greek-alphabet.css:
```diff
@@ -0,0 +1,110 @@
+}
+
+
+.flip__all__button {
```
why do we have `.flip_all_button css` ;  duplicated in various files such as geek-alphabet.css and greek-alphabet.css while this flip feature is already available.

xkeshav commented on src/assets/styles/varnmala/months.css:
```diff
@@ -0,0 +1,425 @@
+}
+
+
+.flip__all__button {
```
again duplicate css

xkeshav commented on src/components/DrawKeyboard.astro:
```diff
@@ -25,95 +38,436 @@ import { IMAGE_DIR, colorBox, fontBox, numberBox } from "@/utils/constants";
+  });
 </script>
 
 <style>
```
move this to css file

xkeshav commented on src/components/Help.astro:
```diff
@@ -54,6 +60,26 @@ const { title, description } = Astro.props;
+  /* Original code */
+
+  /*
+  .help {
```
delete commented code

xkeshav commented on src/components/HideSeekKbd.astro:
```diff
@@ -2,6 +2,7 @@
 import { getLetterIndex } from "@/utils/index";
 
 const letterRows = [
```
same code has been added in keyboard.astro; move this is constants and import from there

xkeshav commented on src/components/keyboard.astro:
```diff
@@ -0,0 +1,132 @@
+</script>
+
+<style>
+.virtual-keyboard {
```
move this to css file

Suresh-Chelani committed:
added andhra pradesh data & update tag color in state statistics

xkeshav commented on src/pages/alphabets/hindi/board.astro:
```diff
@@ -0,0 +1,153 @@
+import BackButton from "@/components/ui/backButton.astro";
+import "@/assets/styles/varnmala/greek-alphabet.css";
+
+const vowels = [
```
move these to constant or fetch it from some methods. no hard code if possible

xkeshav commented on src/pages/glossary/days.astro:
```diff
@@ -0,0 +1,122 @@
+const pageTitle = "Days of the Week";
+const category = "days"; 
+
+const daysData: DayEntry[] = [
```
move it as json and in separate file and import from there. writing this way make this component very rigid

chhaviS04 replied:
done

xkeshav commented on src/pages/glossary/index.astro:
```diff
@@ -0,0 +1,210 @@
+  icon: string;
+}
+
+const glossarySections: GlossarySection[] = [
```
make all pages json in same structure so that we can just pass the data and utilize same component

chhaviS04 replied:
Created common json for all pages under glossary

chhaviS04 committed:
Seperated Css from Astro

xkeshav commented on src/pages/glossary/months.astro:
```diff
@@ -0,0 +1,184 @@
+  { type: "indian", name: "फाल्गुन ", dates: "February 20 - March 20", monthIndex: 1, displayName: "Phalguna", hinduOrdinalText: "Twelfth Hindu Month" },
+];
+
+const nationalHolidaysData = [
```
redundant data ; it will obsolete when year passed. remove and also make changes where using this

xkeshav commented on src/pages/map/index.astro:
```diff
@@ -1,12 +1,173 @@
+  </main>
 </BaseLayout>
+
+<style>
```
exclude the css

Suresh-Chelani replied:
done.

chhaviS04 committed:
Removed Duplicate code of flip card

chhaviS04 commented on src/assets/styles/varnmala/alphabet.css:
```diff
@@ -36,3 +56,64 @@
+  /* Styles for the Help Icon in mobile view */
+  
+ 
+  .your-help-icon-selector { 
```
Sir earlier there is common alphabets with hindi and english switching tabs at that time I adjusted the top menu for mobile view

Azmat01 committed:
Refactor Card and Clan components for improved accessibility and styling for print.

Ayu-Ty committed:
fixed text color bug

chhaviS04 committed:
Created seperated json for glossary page

Suresh-Chelani committed:
refactor: refactor component css

xkeshav committed:
[RZA-250021]: fix lint issues

xkeshav committed:
[RZA-250021]: fix lint issues

Rupesh-Rathore committed:
new

1046prt committed:
Refactor the CSS file for Card and Canvas Page

1046prt committed:
Refactor the css file of keyboard.astro

1046prt committed:
Add DrawPage.css for styling and refactor draw.astro to use external CSS

Ayu-Ty committed:
fixed build issues

chhaviS04 committed:
fixed numbers

Ayu-Ty committed:
fixed back buttons

chhaviS04 commented on src/pages/glossary/months.astro:
```diff
@@ -0,0 +1,184 @@
+const today = new Date(currentYear, 5, 11); 
+
+const englishMonthsData: MonthEntry[] = [
+  { type: "english", name: "January", hindi: "जनवरी", monthIndex: 0 },
```
used Intl method to create months name

Rupesh-Rathore committed:
build issues removed

Rupesh-Rathore committed:
Merge branch 'feature/RZA-250021' of https://github.com/recursivezero/abcd into feature/RZA-250021

Rupesh-Rathore committed:
constants moved to constants.ts

Rupesh-Rathore committed:
files renamed

Rupesh-Rathore committed:
build issue resolved

chhaviS04 committed:
fixed import path

Ayu-Ty committed:
fixes

chhaviS04 committed:
created a seperated json for numbers

chhaviS04 committed:
Removed duplicate code

chhaviS04 committed:
fixed morse code

Ayu-Ty committed:
added back

Rupesh-Rathore committed:
discliamer css modified

1046prt committed:
Update background image path handling based on network status

Suresh-Chelani committed:
added data for states

chhaviS04 committed:
fixed path

Ayu-Ty committed:
fixes

Suresh-Chelani committed:
fix build issue & loading uttarakhand svg

Suresh-Chelani committed:
Remove other categories

Suresh-Chelani committed:
fix alignment of state statistics

chhaviS04 committed:
corrected font colour

Suresh-Chelani committed:
added mobile view

chhaviS04 committed:
created write section in varnmala

chhaviS04 committed:
integrated speech api

Ayu-Ty committed:
back fix

Rupesh-Rathore committed:
constants moved to constants.ts

xkeshav committed:
[RZA-250021]: fix filenames and astro files

Ayu-Ty committed:
text color improvement

1046prt committed:
adjust margin for alphabet header and nav button for better spacing

xkeshav committed:
[RZA-250021]: fix lint issues and</ BaseLayout> placement

chhaviS04 committed:
UPDATED

chhaviS04 committed:
Added mute button

xkeshav committed:
[RZA-250021]: fixed alias path

xkeshav committed:
[RZA-250021]: light-dark
